src/common/AdaptiveTree.h

```diff
@@ -412,8 +412,7 @@ class ATTRIBUTE_HIDDEN AdaptiveTree
   }*current_period_, *next_period_;
 
   std::vector<Period*> periods_;
-  std::string manifest_url_, base_url_, effective_url_, base_domain_, effective_domain_,
-      update_parameter_;
+  std::string manifest_url_, base_url_, effective_url_, base_domain_, update_parameter_;
```

Comment:
Just a comment on this. Creating multiple variables like this is usually best avoided. The reasoning is probably something like, "Oh but those variables are related so I create them on the same line". 

Doing it this way makes it hard to add a comment and someone new to the code doesn't know where they should add their new variable, as a type like this one has several places where multiple vars are created. If multiple groups of separate types are needed then use a function to denote their relationship instead.

BTW, I'm not familiar with the code, maybe there is a very good reason for this.

Reply:
I was just keeping with what was already there. Should I change?

Reply:
I would just define one on each line.

Reply:
Done :)